public/app/features/invites/SignupInvited.tsx

```diff
@@ -32,10 +32,8 @@ const navModel = {
   },
 };
 
-export interface Props extends GrafanaRouteComponentProps<{ code: string }> {}
-
-export const SignupInvitedPage = ({ match }: Props) => {
-  const code = match.params.code;
+export const SignupInvitedPage = () => {
+  const { code } = useParams();
```

Comment:
do we need a default value for this? 🤔

Reply:
Not really, if there's no `code` param, the page won't be rendered as the URL is `/invite/:code`.